src/main/java/org/folio/service/orders/utils/ProductIdUtils.java

```diff
@@ -36,6 +39,11 @@ public static boolean isISBN(String isbnTypeId, ProductId productId) {
     return Objects.equals(productId.getProductIdType(), isbnTypeId);
   }
 
+  public static boolean isISBNValidationException(Throwable throwable) {
+    return throwable instanceof HttpException httpException && Objects.nonNull(httpException.getError())
+      && httpException.getError().getCode().equals(ISBN_NOT_VALID.getCode());
```

Comment:
I am not sure if code can be null, but better to check like this:
ISBN_NOT_VALID.getCode().eqauls(httpException.getError().getCode())

Reply:
done